src/utils/getOssUserType.ts

```diff
@@ -28,7 +27,7 @@ export async function getOssUserType(
     return null;
   }
 
-  const org = owner.login;
+  const orgSlug = owner.login;
```

Comment:
Updating this variable name to match (new) usage throughout the rest of the codebase.